### ISSUE TITLE
sarcoplasmic bodies

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9737]
**Preferred term label:**
Sarcoplasmic bodies

**Synonyms**

**Definition (free text, please give PubMed ID)**
Rounded or oval structures that appeared brown on hematoxylin and eosin and red on the modified Gomori’s trichrome stain

**Parent term (use hpo.jax.org/app)**
UMLS C0333780

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://omim.org/entry/620286

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
added as HP:0034722
